intercept/src/intercept.h

```diff
@@ -1947,10 +1947,15 @@ inline uint64_t CLIntercept::getEnqueueCounter() const
 
 inline uint64_t CLIntercept::incrementEnqueueCounter()
 {
+    uint64_t enqueueCounter = m_EnqueueCounter.load();
+    if( enqueueCounter == m_Config.StopOnEnqueueCount ) {
+        log(">>> Stop enqueue counter " + std::to_string(enqueueCounter) + " reached - terminating application...\n");
```

Comment:
The `>>>` syntax is used in the log to indicate function entry, so we shouldn't use it here.

```suggestion
        log("Stop enqueue counter " + std::to_string(enqueueCounter) + " reached - terminating application...\n");
```

Reply:
Done